compatibility/bazel_tools/daml_sdk.bzl

```diff
@@ -3,7 +3,7 @@
 
 runfiles_library = """
 # Copy-pasted from the Bazel Bash runfiles library v2.
-set -uox pipefail; f=bazel_tools/tools/bash/runfiles/runfiles.bash
+set -uo pipefail; f=bazel_tools/tools/bash/runfiles/runfiles.bash
 source "${{RUNFILES_DIR:-/dev/null}}/$f" 2>/dev/null || \
```

Comment:
The remainder of this snippet uses double braces when it should use single braces. Best take a fresh copy from `BAZEL-bash.md`.

Reply:
Oh good catch, thank you! That’s leftover from when I still had this inlined.